src/hooks/useCreateTargetForm.js

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react';
+import { useStatus } from '@rootstrap/redux-tools';
+
+import { createTarget, getTopics } from 'actions/targetActions';
+
+const useCreateTargetForm = (topic, TARGET, handleSubmit) => {
+  const { error: targetError, status: targetStatus } = useStatus(createTarget);
+  const { error: getTopicsError, status: getTopicsStatus } = useStatus(getTopics);
+  const [topicError, setTopicError] = useState();
+
+  useEffect(() => {
+    setTopicError(null);
+  }, [topic]);
+
+  const handleOnPress = () => {
+    if (!topic) {
+      setTopicError(TARGET.topicError);
+    } else {
+      setTopicError(null);
+    }
```

Comment:
What about a ternary operator?
```suggestion
    topic ? setTopicError(TARGET.topicError) : setTopicError(null)
```